content/collections/audit-works.html

```diff
@@ -1,3 +1,315 @@
+<item>
+  <field name="logo">
+    thirdweb.png
+  </field>
+  <field name="title">
+    thirdWeb-15
+  </field>
+  <field name="subtitle">
+    Web3 developer tooling
+  </field>
+  <field name="date">
+    2023-10-06
+  </field>
+  <field name="about">
+
+  </field>
+  <field name="report">
+    /library/audits/thirdweb-15
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/thirdweb-dev/contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    thirdweb.png
+  </field>
+  <field name="title">
+    thirdWeb-14
+  </field>
+  <field name="subtitle">
+    Web3 developer tooling
+  </field>
+  <field name="date">
+    2023-08-21
+  </field>
+  <field name="about">
```

Comment:
Audited the EvolvingNFT and LoyaltyPoints contracts, as well as changes to the smart contract wallet account contracts and their factories.